cfme/fixtures/v2v_fixtures.py

```diff
@@ -29,6 +30,29 @@
 V2vProviders = namedtuple("V2vProviders", ["vmware_provider", "rhv_provider", "osp_provider"])
 
 
+def set_skip_event_history_flag(appliance):
+    """This flag is required for OSP to skip all old events and refresh inventory"""
+    appliance.update_advanced_settings(
+        {'ems': {'ems_openstack': {'event_handling': {'event_skip_history': True}}}})
+    appliance.evmserverd.restart()
+    appliance.evmserverd.wait_for_running()
+    appliance.wait_for_web_ui()
+
+
+def start_event_workers_for_osp(appliance, provider):
+    """This is a workaround to start event catchers until BZ 1753364 is fixed"""
+    provider_edit_view = navigate_to(provider, 'Edit')
+    endpoint_view = provider.endpoints_form(parent=provider_edit_view)
+    endpoint_view.events.event_stream.select("AMQP")
```

Comment:
could you please explain why you set  event stream two times ?

Reply:
@izapolsk: this is workaround for openstack eventcatcher workers not getting started. You can read this devel's explanation https://bugzilla.redhat.com/show_bug.cgi?id=1715455#c13.

This should be fixed by https://bugzilla.redhat.com/show_bug.cgi?id=1753364.

Reply:
Can we actually wrap this logic in a check against `BZ(1753364).blocks` please?

Automation should stop running it ASAP once a build fixes it, instead of an update to the framework lagging behind the fix.  Test status before/after the BZ is fixed should reflect whether the BZ can be validated.

If the bug no longer blocks, log that and assert that the workers are running.

Reply:
And now I scroll down and see where this function is actually used within the fixture.

Can you change this function to be named `_start_event_workers_for_osp`, indicating its local-only intended use.

Even better, standardize its failure mode more - currently it could raise an AssertionError in setup, or skip. I think it should more consistently raise an AssertionError, and the call to pytest.skip should be in an except statement in the fixture method itself, not here in the module function.